forms/forms-view/src/main/java/org/bonitasoft/forms/client/view/controller/FormPagesViewController.java

```diff
@@ -392,9 +406,29 @@ protected void buildPage(final ReducedFormPage formPage, final boolean hasAlread
         }
         domUtils.overrideBrowserNativeInputs();
         domUtils.hideLoading();
+        focusFirstField();
+
+    }
+
+    /**
+     * Focus the first field "focusable"
+     */
+    protected void focusFirstField() {
+        final Iterator<Entry<String, FormFieldWidget>> fieldWidgestIterator = fieldWidgets.entrySet().iterator();
+
+        boolean focused = false;
+        while (fieldWidgestIterator.hasNext() && !focused) {
+            final Entry<String, FormFieldWidget> reducedFormWidget = fieldWidgestIterator.next();
+            final FormFieldWidget formFieldWidget = reducedFormWidget.getValue();
+            if (formFieldWidget!=null &&  formFieldWidget.getType()!=null && widgetTypesToFocus.contains(formFieldWidget.getType())) {
```

Comment:
- indentation to fix
- I think the second null check is nor necessary (the type can never be null and event if if was, there is no null value in widgetTypesToFocus)